cmd/storage.go

```diff
@@ -65,17 +86,24 @@ func initStorageCmd() {
 		Use:   "storage",
 		Short: "Operations on storages",
 		Long:  `List, create, or remove storages.`,
-		Run:   produceStorageCmdRunFunc(client),
 	}
 
 	var storageLsCmd = &cobra.Command{
 		Use:     "ls [flags]",
 		Aliases: []string{"list"},
 		Short:   "List storages",
 		Long:    `List storage objects.`,
-		Run:     produceStorageCmdRunFunc(client),
+		Run:     produceStorageCmdRunFunc(client, storageListAction),
+	}
+	var removeCmd = &cobra.Command{
+		Use:     "rm [flags] [ID]",
+		Aliases: []string{"remove"},
+		Short:   "Remove Storage",
+		Long:    `Remove an existing Storage.`,
```

Comment:
Same here. Both "Storage" please lowercase.

Reply:
Oops, of course 😃